patches/api/0480-add-Plugin-getDataPath.patch

```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: powercas_gamer <cas@mizule.dev>
+Date: Mon, 15 Jul 2024 02:07:49 +0000
+Subject: [PATCH] add Plugin#getDataPath
+
+
+diff --git a/src/main/java/org/bukkit/plugin/Plugin.java b/src/main/java/org/bukkit/plugin/Plugin.java
+index 61a9192ae1bf676e32c961b3bef920f7f3f48867..b89de42bae2fd0f70e7c54c3ea103c90df338397 100644
+--- a/src/main/java/org/bukkit/plugin/Plugin.java
++++ b/src/main/java/org/bukkit/plugin/Plugin.java
+@@ -18,7 +18,7 @@ import org.jetbrains.annotations.Nullable;
+  */
+ public interface Plugin extends TabExecutor, io.papermc.paper.plugin.lifecycle.event.LifecycleEventOwner { // Paper
+     /**
+-     * Returns the folder that the plugin data's files are located in. The
++     * Returns the folder that the plugin data files are located in. The
```

Comment:
Wrong patch?

Reply:
fixed. Saw this when I was making the getDataPath() and later edited the other patch